skt/executable.py

```diff
@@ -928,6 +934,12 @@ def load_config(args):
         cfg['publisher'] = [config.get('publisher', 'type'),
                             config.get('publisher', 'destination'),
                             config.get('publisher', 'baseurl')]
+        if config.has_option('publisher', 'prefix'):
+            cfg['publisher'].append(config.get('publisher', 'prefix'))
+        else:
+            cfg['publisher'].append('')
+    elif cfg.get('publisher'):
+        cfg['publisher'].append(cfg['prefix'])
```

Comment:
It's hard to figure out what's going on here. Do you think we can rename these variables eventually?

Reply:
see the new issue #350 which I submitted when I saw how the configuration is parsed

Reply:
I think #350 is spot on, but I was talking about the `cfg` vs. `config` variable names here, and how difficult it is to make sense of which means what.

Reply:
Ah, ok. I agree that it's confusing. Maybe submit another issue for clear variable naming if there isn't one?

Reply:
Done: #351 :)